logging/request_context_logger.go

```diff
@@ -0,0 +1,70 @@
+package logging
+
+import (
+	"github.com/giantswarm/request-context"
+)
+
+// NewRequestContextLogger returns a RequestContextLogger, given a Config.
+func NewRequestContextLogger(config Config) RequestContextLogger {
```

Comment:
I didn't want to expose `request-context` outside of the interface. This wraps it all up.